libraries/core-react/src/components/Popover/Popover.tsx

```diff
@@ -1,101 +1,162 @@
 import * as React from 'react'
-import {
-  forwardRef,
-  useRef,
-  HTMLAttributes,
-  ReactNode,
-  isValidElement,
-} from 'react'
-import styled from 'styled-components'
-import { PopoverItem } from './PopoverItem'
-import { PopoverAnchor } from './PopoverAnchor'
+import { forwardRef, useRef, useState, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Icon } from '../Icon'
+import { Paper } from '../Paper'
+import { Button } from '../Button'
+import { close } from '@equinor/eds-icons'
+import { spacingsTemplate, typographyTemplate } from '@utils'
+import { usePopper, useOutsideClick } from '@hooks'
+import { popover as tokens } from './Popover.tokens'
 
-const Container = styled.div`
-  position: relative;
-  display: inline-flex;
-  justify-content: center;
+type StyledPopoverProps = Pick<PopoverProps, 'open'>
+
+const StyledPopover = styled(Paper)<StyledPopoverProps>`
+  ${typographyTemplate(tokens.header)}
+  ${spacingsTemplate(tokens.spacings)}
+  display: grid;
+  grid-gap: 16px;
+  grid-auto-columns: auto;
+  align-items: center;
+  align-content: start;
+  background: ${tokens.background};
+  width: max-content;
+  max-height: ${tokens.popover.maxHeight};
+  max-width: ${tokens.popover.maxWidth};
+  min-height: ${tokens.popover.minHeight};
+  z-index: 100;
+
+  ${({ open }) =>
+    css({
+      visibility: open ? 'visible' : 'hidden',
+    })};
+
+  .arrow {
+    z-index: -1;
+    width: 10px;
+    height: 10px;
+  }
+  &[data-popper-placement^='top'] > .arrow {
+    bottom: -4px;
+  }
+
+  &[data-popper-placement^='bottom'] > .arrow {
+    top: -4px;
+  }
+
+  &[data-popper-placement^='left'] > .arrow {
+    right: -4px;
+  }
+
+  &[data-popper-placement^='right'] > .arrow {
+    left: -4px;
+  }
 `
 
-const Anchor = styled.div`
-  &:focus {
-    outline: none;
+const StyledCloseButton = styled(Button)`
+  position: absolute;
+  top: 8px;
+  right: 16px;
+  height: 32px;
+  width: 32px;
```

Comment:
Same as above, can we use token here to clarify what this is? 🤔  It would be a lot easier to conform on `ComponentType` later on